docs/samples/Microsoft.ML.Samples/Dynamic/Transforms/Projection/VectorWhitenWithColumnInfo.cs

```diff
@@ -0,0 +1,62 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using Microsoft.ML.Data;
+
+namespace Microsoft.ML.Samples.Dynamic
+{
+    public sealed class VectorWhitenWithColumnInfo
+    {
+        /// <summary>
+        /// This example require installation of addition nuget package <a href="https://www.nuget.org/packages/Microsoft.ML.HalLearners/">Microsoft.ML.HalLearners</a>
+        /// </summary>
+        public static void Example()
+        {
+            // Create a new ML context, for ML.NET operations. It can be used for exception tracking and logging, 
+            // as well as the source of randomness.
+            var ml = new MLContext();
+
+            // Get a small dataset as an IEnumerable and convert it to an IDataView.
+            var data = SamplesUtils.DatasetUtils.GetVectorOfNumbersData();
+            var trainData = ml.Data.ReadFromEnumerable(data);
+
+            // Preview of the data.
+            //
+            // Features
+            // 0   1   2   3   4   5   6   7   8   9
+            // 1   2   3   4   5   6   7   8   9   0  
+            // 2   3   4   5   6   7   8   9   0   1
+            // 3   4   5   6   7   8   9   0   1   2
+            // 4   5   6   7   8   9   0   1   2   3
+            // 5   6   7   8   9   0   1   2   3   4
+            // 6   7   8   9   0   1   2   3   4   5
+
+            // A small printing utility.
+            Action<string, IEnumerable<VBuffer<float>>> printHelper = (colName, column) =>
+            {
+                Console.WriteLine($"{colName} column obtained post-transformation.");
+                foreach (var row in column)
+                    Console.WriteLine($"{string.Join(" ", row.DenseValues().Select(x => x.ToString("f3")))} ");
+            };
+
+
+            // A pipeline to project Features column into white noise vector.
+            var whiteningPipeline = ml.Transforms.Projection.VectorWhiten(new Transforms.Projections.VectorWhiteningEstimator.ColumnInfo(
+                nameof(SamplesUtils.DatasetUtils.SampleVectorOfNumbersData.Features), kind: Transforms.Projections.WhiteningKind.Pca, pcaNum:4));
+            // The transformed (projected) data.
+            var transformedData = whiteningPipeline.Fit(trainData).Transform(trainData);
+            // Getting the data of the newly created column, so we can preview it.
+            var whitening = transformedData.GetColumn<VBuffer<float>>(ml, nameof(SamplesUtils.DatasetUtils.SampleVectorOfNumbersData.Features));
+
+            printHelper(nameof(SamplesUtils.DatasetUtils.SampleVectorOfNumbersData.Features), whitening);
+
+            // Features column obtained post-transformation.
+            // -0.979  0.867  1.449  1.236
```

Comment:
>0 [](start = 23, length = 1)

I think this is a vector column, if so, could you use senja's suggestion of printing the vector inside brackets: [....]